masl/test/9441_subsupernavigation/test.svc

```diff
@@ -8,4 +8,8 @@ begin
   dog     := animal->R1;
   animals := dogs->R1; 
   dogs    := animals->R1;
+  animal  := dog->R1.Animal;  // Failure at this line
```

Comment:
Assuming this works now, remove the comment here and few lines down about the failure